src/commands/emasser/get/artifacts.ts

```diff
@@ -13,7 +13,7 @@ import {FlagOptions,
 const endpoint = 'artifacts'
 
 export default class EmasserGetArtifacts extends Command {
-  static usage = '<%= command.id %> [ARGUMENTS]';
+  static usage = '<%= command.id %> [ARGUMENT] \n \x1B[93m NOTE: see EXAMPLES for argument case format\x1B[0m';
```

Comment:
Which command makes this show up?

Reply:
bin/run emasser get artifacts -h or --help